specs/centos_ci.adoc

```diff
@@ -2,28 +2,46 @@
 
 = CentOS CI
 
-CentOS CI should be configured to launch applicable set of tests as an
```

Comment:
I think there should be some "Introduction paragraph" (see the sample_template.adoc)